options/options.html

```diff
@@ -49,11 +49,14 @@
           <div>
             <label><input id="linkReddit" type="checkbox">Convert subreddit names outside of reddit.com</label>
           </div>
+          <div>
+            <label><input id="linkOnPre" type="checkbox">Turn link in &lt;pre&gt;</label>

```

Comment:
Can we reword the description to: "Convert links inside preformatted text blocks (\<pre> tag)"

Hopefully it'll help non-devs understand what the option does a little better :)